api/models/error.go

```diff
@@ -51,6 +53,11 @@ func (m *Error) validateMessage(formats strfmt.Registry) error {
 	return nil
 }
 
+// ContextValidate validates this error based on context it is used
```

Comment:
```suggestion
// ContextValidate validates this error based on context it is used in
```
word missing? same for other occurences

Reply:
the file(s) are generated by the swagger tool ¯\_(ツ)_/¯